docs/package.json

```diff
@@ -51,6 +51,7 @@
     "core-js": "^2.6.12",
     "cross-env": "^7.0.3",
     "date-fns-jalali": "^2.21.3-1",
+    "dayjs": "^1.11.6",
```

Comment:
This PR targets `next` branch, but at least I can't see the same issue in [next docs](https://next.mui.com/x/react-date-pickers/localization/).
Have you looked into why this could be? 🤔

Reply:
I guess it is a good thing to add it to the `package.json` anyway
But it should definitely back-ported to master.